src/graphql.js

```diff
@@ -105,6 +285,9 @@ const queryRequest = ({token, query, debug, dryRun, verbose, name}) => {
       'User-Agent': 'Name-Your-Contributors',
       'Authorization': `bearer ${token}`
     }
+
+    const runQ = queryCost(query, dryRun)
```

Comment:
Not sure why runQ calls queryCost. Kind of confusing.

Reply:
`queryCost` wraps a query in another query that also asks the cost (and optionally sets the `dryRun` flag). I could rename it something like `queryWithCost`.

Reply:
That would be a bit more useful, I think.